tests/test_config.py

```diff
@@ -0,0 +1,186 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Oxford Quantum Circuits Ltd
+from os.path import abspath, dirname, join
+from sys import __loader__
+
+import pytest
+
+from compiler_config.config import (
+    CompilerConfig,
+    InlineResultsProcessing,
+    MetricsType,
+    OptimizationConfig,
+    Qasm2Optimizations,
+    QiskitOptimizations,
+    QuantumResultsFormat,
+    ResultsFormatting,
+    TketOptimizations,
+)
+
+SUPPORTED_CONFIG_VERSIONS = ["v02", "v01", "v1"]
+
+
+class TestType:
+    pass
+
+
+def _get_json_path(file_name):
```

Comment:
It can make sense to move common things into conftest.py so that the file just contains tests, and these functions can be re-used